salt/cloud/clouds/saltify.py

```diff
@@ -74,3 +79,46 @@ def get_configured_provider():
         __active_provider_name__ or 'saltify',
         ()
     )
+
+
+def _verify(vm_):
+    '''
+    Verify credentials for an exsiting system
+    '''
+    log.info('Testing logon credentials for {0}'.format(vm_['name']))
+
+    win_installer = config.get_cloud_config_value(
+        'win_installer', vm_, __opts__
+    )
+
+    if win_installer:
```

Comment:
@cachedout Yes, that is the way to check if salt-cloud is attempting to provision a Windows system. For Windows + salt-cloud, you set `win_installer` in your salt-cloud configuration. [Here's an example](https://docs.saltstack.com/en/latest/topics/cloud/windows.html#configuration).